pkgs/tools/security/clairvoyance/default.nix

```diff
@@ -5,14 +5,14 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "clairvoyance";
-  version = "2.0.6";
+  version = "2.5.3";
```

Comment:
The title of the PR and the commit message seems wrong, shouldn't it have been `clairvoyance: 2.0.6 -> 2.5.3`

Reply:
Upp, it was wrong. Thanks.